dask_jobqueue/pbs.py

```diff
@@ -60,66 +62,44 @@ class PBSCluster(JobQueueCluster):
 
     >>> cluster.adapt()
     """
+
+    _submitcmd = 'qsub'
+    _cancelcmd = 'qdel'
```

Comment:
It might be nice to rename these to the more explicit (if perhaps also longer) `submit_command` and `cancel_command` and remove the properties that point to them.

Reply:
Okay, I'll try this, comming from Java background, it was more natural to me to override kind of methods instead of attributes :).